packages/cli/src/commands/generate/dbAuth/dbAuth.js

```diff
@@ -114,9 +124,21 @@ export const files = ({
   skipReset,
   skipSignup,
   webAuthn,
+  username,
+  password,
 }) => {
   const files = []
 
+  const usernameLowercase =
+    username == null ? 'username' : username.toLowerCase()
```

Comment:
```suggestion
  const usernameLowercase = username?.toLowerCase() || 'username'
```

Reply:
Here's another way to do it. You decide which one you prefer 🙂 

```suggestion
  const usernameLowercase = (username || 'username').toLowerCase()
```